todo/src/components/Form.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import {addTask} from '../actions';
+import './todo.css'
+
+class Form extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            todo: ''  
+    }}
+
+    handleInputChange = e => {
+        this.setState({todo: e.target.value})
+      };
+
+      addTodo = e => {
+          e.preventDefault();
+          this.props.addTask(this.state.todo)
+          this.setState({todo: ''})
+          document.querySelector('#input').value = ''
+      }
+
+    render() {
+        return (
+        <div className="form-container">
+            <form onSubmit={this.addTodo} autoComplete="off" className="todo-form">
+            <input
+                onChange={this.handleInputChange}
+                placeholder="Task"
+                value={this.state.name}
+                name="name"
+                type="text"
+                id="input"
+                required
+            />
+            
+            <button type="submit">Add Task</button>
+            </form>
+        </div>
+        )
+    }
+}
+
+const mapStateToProps = state => {
+    return {
+      todos: state.todos
+    };
```

Comment:
Why map the `todos` variable to this component's `props`? It is actually not being used, so you could forgo this function and replace `mapStateToProps` with `null` in your `connect` function.